src/components/heading/heading.js

```diff
@@ -20,6 +21,7 @@ export default class Heading extends PureComponent {
     style: {},
     renderAs: 'h1',
     size: null,
+    weight: 'normal',
```

Comment:
please use null as a default so by default the component will not set has-text-weight-normal as a default value